src/editor/Editor.js

```diff
@@ -2751,35 +2985,35 @@ define(function (require, exports, module) {
      * line numbers are hidden.
      * @param {boolean} showLinePadding
      */
-    Editor._toggleLinePadding = function (showLinePadding) {
+    Editor._toggleLinePadding = function(showLinePadding) {
         // apply class to all pane DOM nodes
         var $holders = [];
-        _instances.forEach(function (editor) {
+        _instances.forEach(function(editor) {
             var $editorHolder = editor.$el.parent();
             if ($holders.indexOf($editorHolder) === -1) {
                 $holders.push($editorHolder);
             }
         });
 
-        _.each($holders, function ($holder) {
+        _.each($holders, function($holder) {
             $holder.toggleClass("show-line-padding", Boolean(showLinePadding));
         });
     };
-    
+
     Editor.LINE_NUMBER_GUTTER_PRIORITY = LINE_NUMBER_GUTTER_PRIORITY;
     Editor.CODE_FOLDING_GUTTER_PRIORITY = CODE_FOLDING_GUTTER_PRIORITY;
-    
+
     // Set up listeners for preference changes
-    editorOptions.forEach(function (prefName) {
-        PreferencesManager.on("change", prefName, function () {
-            _instances.forEach(function (editor) {
+    editorOptions.forEach(function(prefName) {
+        PreferencesManager.on("change", prefName, function() {
+            _instances.forEach(function(editor) {
                 editor._updateOption(prefName);
             });
         });
     });
 
     // Define public API
-    exports.Editor                  = Editor;
-    exports.BOUNDARY_CHECK_NORMAL   = BOUNDARY_CHECK_NORMAL;
-    exports.BOUNDARY_IGNORE_TOP     = BOUNDARY_IGNORE_TOP;
+    exports.Editor = Editor;
+    exports.BOUNDARY_CHECK_NORMAL = BOUNDARY_CHECK_NORMAL;
+    exports.BOUNDARY_IGNORE_TOP = BOUNDARY_IGNORE_TOP;
```

Comment:
Most of the changes here are formatting the file, but they are not necessary and will cause trouble for other developers while rebasing their repositories, for example. Just keep it simple and focus in your changes.